outputs/http.go

```diff
@@ -0,0 +1,67 @@
+package outputs
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/aquasecurity/postee/layout"
+)
+
+type HTTPClient struct {
+	Name    string
+	Client  http.Client
+	URL     *url.URL
+	Method  string
+	Body    string
+	Headers map[string][]string
+}
+
+func (hc *HTTPClient) GetName() string {
+	return hc.Name
+}
+
+func (hc *HTTPClient) Init() error {
+	hc.Name = "HTTP Output"
+	return nil
+}
+
+func (hc HTTPClient) Send(m map[string]string) error {
+	headers := hc.Headers
```

Comment:
this is better but some clone operation is required here. Otherwise no logic is changed (it's just same link)
like:
```golang
// Create the target map
targetMap := make(map[string]int)

// Copy from the original map to the target map
for key, value := range originalMap {
  targetMap[key] = value
}
```

Reply:
something similar is required for `ExecClient.Env`

Reply:
I see - thats a good catch. I updated both here: https://github.com/aquasecurity/postee/pull/249/commits/c115a0fad4647e4649be9647d1dd33a0ef0e00a8 and https://github.com/aquasecurity/postee/pull/249/commits/87ce03738ee1dd1501efbf424c4691e181687468